Nonprofit-Exchange-Hub/web-app
# add eslint to server and prettierify server

## Dev Summary

this PR is being compared to branch from https://github.com/Nonprofit-Exchange-Hub/web-app/pull/86
this PR should wait until after that PR is reviewed/landed

- moved old eslint file (removed in https://github.com/Nonprofit-Exchange-Hub/web-app/pull/86) into server
- ran `npm run format`
  - which changed predominantly whitespace throughout server


## Test Plan


1. `cd server && npm run checks`
2. change auth.controller to have...
```
  constructor(private authService: AuthService) {
    
  }
```
3. run `npm run checks`
see that we get an error and that eslint is therefore working


## Resources

- https://www.youtube.com/watch?v=bfyI9yl3qfE
- https://eslint.org/docs/rules/
